components/turret.py

```diff
@@ -206,6 +209,10 @@ def is_ready(self) -> bool:
         )
 
     def execute(self) -> None:
+        if self.disabled:
+            self.motor.stop()
```

Comment:
Isn't the method called `disable` or `stopMotor`?

Reply:
Yes. Done in a bus to Canberra with no internet.